tests/phpunit/FunctionalTest.php

```diff
@@ -138,6 +138,7 @@ public function testTestConnection()
     {
         $this->initConfig(function ($config) {
             $config['action'] = 'testConnection';
+            unset($config['parameters']['tables']);
```

Comment:
to ma fungovat i kdyz tam tables jsou diky tomy `ignoreExtraKeys` zejo?

Reply:
jj mělo by to fungovat